docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -26,6 +26,10 @@ If you are on Python 2, you'll first need to install a backport of the
 :ref:`running-unit-tests-dependencies` for details on installing `mock`_ and
 the other optional test dependencies.
 
+You also need to install pytz_ regardless of your Python version::
+
+   $ pip install pytz
```

Comment:
Have done this in https://github.com/django/django/pull/7437. I'll merge that first if it looks fine.